java-vtl-model/src/main/java/no/ssb/vtl/model/VTLObject.java

```diff
@@ -83,32 +68,29 @@ public Object get() {
     }
 
     /**
-     * Do not use!
+     * Create a new VTLString instance.
+     *
+     * @throws NullPointerException if str is null.
      */
-    @Deprecated
-    public static VTLObject of(Component component, Object o) {
-        if (o == null)
-            return VTLObject.NULL;
-
-        if (o instanceof VTLObject) {
-            return (VTLObject) o;
-        }
-
-        return new VTLObject<Object>(component){
+    public static VTLString of(String str) {
+        return VTLString.of(str);
+    }
 
-            @Override
-            public Object get() {
-                return o;
-            }
-        };
+    /**
+     * Create a new VTLBoolean instance.
+     *
+     * @throws NullPointerException if str is null.
```

Comment:
Not really. I thought of using a single static instance of VTLObject as NULL (VTLObject.NULL) but is does not work so well in the end. Maybe having a "isNull()" method would be a better approach?
What I tried to avoid here is tests like 
```java
VTLObject nullable = VTLObject.of("String")
if (nullable != null && nullable.get() != null) {
  // ...
}
```